packages/blade/src/components/ThemeProvider/__tests__/ThemeProvider.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { renderHook, act, WrapperComponent } from '@testing-library/react-hooks';
+import { paymentsTheme, colorSchemeNamesInput } from '../../../tokens/theme';
+import { ThemeProvider, useTheme, ThemeProviderProps } from '../../ThemeProvider';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+describe('<ThemeProvider/>', () => {
+  const wrapper: WrapperComponent<ThemeProviderProps> = ({ theme, colorScheme, children }) => (
+    <ThemeProvider theme={theme} colorScheme={colorScheme}>
+      {children}
+    </ThemeProvider>
+  );
+
+  it('should render with provided theme and colorscheme', () => {
+    const { result } = renderHook(() => useTheme(), {
+      wrapper,
+      initialProps: {
+        theme: paymentsTheme,
+        colorScheme: 'light',
+        children: null,
+      },
+    });
+
```

Comment:
you might wonder what is this wrapper and initialprops so this is how we render context providers using `react-hooks` package. [Reference documentation](https://react-hooks-testing-library.com/usage/advanced-hooks)